Mathlib/Probability/Moments.lean

```diff
@@ -285,6 +286,21 @@ theorem iIndepFun.cgf_sum {X : ι → Ω → ℝ}
   · rw [h_indep.mgf_sum h_meas]
   · exact (mgf_pos (h_int j hj)).ne'
 
+theorem mgf_congr_identDistrib
+    (X : Ω → ℝ) {Ω' : Type*} {m' : MeasurableSpace Ω'} {μ' : Measure Ω'} (X' : Ω' → ℝ)
+    (hident : IdentDistrib X X' μ μ') (t : ℝ) :
+    mgf X μ t = mgf X' μ' t := hident.comp (measurable_const_mul t).exp |>.integral_eq
+
+theorem mgf_sum_of_identDistrib
+    {X : ι → Ω → ℝ}
+    (h_meas : ∀ (i : ι), Measurable (X i))
```

Comment:
But `hident` does not imply `Measurable (X i) μ`, right? And we need it for `mgf_sum`. 
A later refactor might weaken the measurability assumption of `mgf_sum` to `AEMeasurable` or `AEStronglyMeasurable` instead of `Measurable`. But for now, I need `Measurable (X i) μ`.

Reply:
Ugh, that's annoying, you're right. Can you leave a TODO then?